scripts/prepare-circle-integration-tests.sh

```diff
@@ -28,4 +28,6 @@ kubectl apply -f k8s/conf/crd-networkservices.yaml
 kubectl apply -f k8s/conf/crd-networkserviceendpoints.yaml
 kubectl apply -f k8s/conf/crd-networkservicemanagers.yaml
 
+# export NSM_NAMESPACE="nsm-system-integration"
```

Comment:
If this is just a hint: prepend it with a proper comment when to enable this line.